File: 백준/Bronze/5893. 17배/17배.kt
import java.math.BigInteger
fun main(){
    val n=readln()
    var num=BigInteger("0")
    for(i in n){
        num*=BigInteger("2")
        if(i=='1'){
            num+=BigInteger("1")
        }
    }
    num*=BigInteger("17")
    var ans=""
    while(num>BigInteger("0")){
        ans = if(num%BigInteger("2")==BigInteger("1")){
            "1$ans"
        } else{
            "0$ans"
        }
        num/= BigInteger("2")
    }
    println(ans)
}

File: 백준/Bronze/5893. 17배/17배.py
n=input()
num=0
for i in range(len(n)):
    num*=2
    if n[i]=='1':
        num+=1
num*=17
ans=""
while num>0:
    if num%2==1:
        ans="1"+ans
    else:
        ans="0"+ans
    num//=2
print(ans)